lib/builder.js

```diff
@@ -110,15 +111,19 @@ module.exports = class Builder extends EventEmitter {
             .then(() => this.emit('beginNode', nw))
             .then(() => nw.build()),
           () => {
-            if (this._cancelationRequest.isCancelled) {
+            if (this._cancelationRequest.isCanceled) {
               return;
             }
             this.emit('endNode', nw);
```

Comment:
Shouldn't `endNode` still be called even in a cancel scenario? When cancelling, the current node should finish, and then no other nodes should be build. This early return above also causes the heimdall issue of incorrect open/close pairs of heimdall start/stop.

Reply:
ya, lets do that. That will prevent some of the unbalanced issues.

Reply:
fixed